gradle/libs.versions.toml

```diff
@@ -203,7 +204,7 @@ google-ksp = { module = "com.google.devtools.ksp:symbol-processing-gradle-plugin
 hamcrest = "org.hamcrest:hamcrest-core:2.2"
 
 java-diff-utils = { module = "io.github.java-diff-utils:java-diff-utils", version.ref = "java-diff-utils" }
-
+telephoto = { module = "me.saket.telephoto:zoomable", version.ref = "telephoto" }
```

Comment:
Was this usage via @saket 's suggestion? :). If not, he should know!

Reply:
yay, love seeing more usage inside our codebase! 🎉

> I briefly tried doing that but it wasn't panning

that might be because compose multiplatform has pretty limited support for zoom events on the JVM :/ last time I checked, I wasn't seeing any trackpad or mouse scroll events. I'll double-check this today. could you share instructions for opening the trace viewer?

Reply:
If you have the workflow kotlin repo cloned you can launch the trace viewer using ./gradlew :workflow-trace-viewer:run. I'm not sure if we have any sample traces in open source but you can download them from any failed test in ci I'll dm you a link.

Reply:
done! https://github.com/square/workflow-kotlin/pull/1421